src/util/HeaderUtil.ts

```diff
@@ -547,3 +552,39 @@ export function parseLinkHeader(link: string | string[] = []): LinkEntry[] {
   }
   return links;
 }
+
+/**
+ * Checks if the value of an HTTP Authorization header matches a specific scheme (e.g. Basic, Bearer, etc).
+ *
+ * @param authorization - The value of the Authorization header (may be undefined).
+ * @param scheme - Name of the authorization scheme (case insensitive).
+ * @returns True if the Authorization header uses the specified scheme, false otherwise.
+ */
+export function matchesAuthorizationScheme(authorization: string | undefined, scheme: string): boolean {
```

Comment:
```suggestion
export function matchesAuthorizationScheme(authorization?: string, scheme: string): boolean {
```
And then the linter is going to complain that optional arguments should come last so you'll have to switch those two.